tests/LineBot.Tests/RichMenuRequestTests/TheNamePropertyTests.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright 2017-2018 Dirk Lemstra (https://github.com/dlemstra/line-bot-sdk-dotnet)
```

Comment:
This file should be renamed to `tests/LineBot.Tests/RichMenu/RichMenuTests/TheNameProperty.cs`.